ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact-3node-cluster/deprovision/cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-deprovision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-deprovision
+  steps:
+  - chain: gather
+  - chain: gather-core-dump
+  - ref: ipi-deprovision-aws-diags
+  - ref: ipi-deprovision-deprovision
+  - ref: ipi-deprovision-aws-dns
+  - ref: ipi-deprovision-aws-lb
```

Comment:
there is no `ipi-deprovision-aws-dns`, `ipi-deprovision-aws-lb` and `ipi-deprovision-aws-diags` steps.

Reply:
you can refer to other `-deprovision` chain to see what steps should be added.